index.html

```diff
@@ -992,12 +992,11 @@ <h3>
           Privacy and security considerations
         </h3>
         <p>
-          When the <a>fullscreen</a> <a>display mode</a> is applied, it is
-          RECOMMENDED that the user agent provides the end-user a means to
-          consult common information about the web application, such as the
-          origin, start and/or current URL, granted permissions, and associated
-          icon. How such information is exposed to end-users is left up to
-          implementers.
+          When the web application is running, it is RECOMMENDED that the user
+          agent provides the end-user a means to consult common information
```

Comment:
nit: maybe s/consult/access?

Reply:
I didn't change that word in this PR, just reformatted the line. But OK while I'm here I'll change it.